functional-tests/notify-suppliers-whether-application-made-for-framework/suppliers.t

```diff
@@ -0,0 +1,51 @@
+
+  $ . $TESTDIR/setup.sh
+
+Supplier ID from command line
+
+  $ ./scripts/notify-suppliers-whether-application-made-for-framework.py --dry-run $DM_ENVIRONMENT g-cloud-10 $NOTIFY_API_KEY --supplier-id=93271
```

Comment:
Can I request we be super cautious and only allow `$DM_ENVIRONMENT` for preview/staging?

Reply:
Sure

Reply:
Added a new commit for this